openstack/master/main.tf

```diff
@@ -49,6 +49,18 @@ resource "null_resource" "generate-inventory" {
 
 }
 
+# Upload all additional public keys to master node
```

Comment:
yeah sure, right now it is only fixed for OpenStack. Perhaps this should be added to the other providers as well? (I do not have access to test this however). But the README.md could be useful yes